### DISCUSSION TITLE
Is there a way to dynamically select a Character?

### DISCUSSION THREAD

**Discussion by nstryder** [dialogic-godot/dialogic#703] [Q&A]
Hello, I'm really liking Dialogic and I was wondering if it is possible to do this:

My game lets you select 3 different characters to be the Player. Each of these 3 characters all share the same set of expressions (Happy, Sad, Angry, etc) but each character uses their own image to portray these. Ideally I'd basically have a Player placeholder in the Timeline editor, and then substitute that placeholder for whatever character the user chose at runtime. 

Is this doable in some way?

**Comment by coppolaemilio** [comment]
It isn't at the moment. One solution I can think is making all those portraits live inside the same character and add a prefix for each variation. But there isn't any native way of doing it
